vm_env/Cargo.toml

```diff
@@ -7,10 +7,18 @@ resolver = "2"
 version = "0.7.1"
 
 [features]
-default = []
+default = ["abciplus"]
+
+abciplus = [
+  "namada/abciplus",
+]
+
+abcipp = [
+  "namada/abcipp",
+]
```

Comment:
I think we should use something other than `abciplus`/`abcipp` for the feature flag names that is more standard. Maybe `tendermint0.37` and `tendermint0.38` respectively, or `abci0.18` and `abci0.19` (although I'm not sure if that new ABCI semver terminology is official yet)